Stream_support/include/CGAL/IO/OM.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2024 GeometryFactory
+//
+// This file is part of CGAL (www.cgal.org);
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: LGPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Andreas Fabri
+
+#ifndef CGAL_IO_OM_H
+#define CGAL_IO_OM_H
+
+#include <OpenMesh/Core/IO/MeshIO.hh>
+#include <OpenMesh/Core/Mesh/TriMesh_ArrayKernelT.hh>
+
+#include <CGAL/boost/graph/graph_traits_PolyMesh_ArrayKernelT.h>
+#include <CGAL/boost/graph/iterator.h>
+#include <CGAL/property_map.h>
+#include <iostream>
+#include <fstream>
+#include <map>
+
+namespace CGAL {
+namespace IO {
+
+template <typename SM, typename VFeaturePM, typename EFeaturePM>
+bool read_OM(std::string fname, SM& sm, VFeaturePM vfpm, EFeaturePM efpm)
```

Comment:
is OM clear enough?
Should it be "OpenMesh" ?

Reply:
We usually put the file extension in the name, don't we?

Reply:
right